src/Kiota.Builder/Kiota.Builder.csproj

```diff
@@ -47,7 +47,7 @@
     <PackageReference Include="Microsoft.OpenApi" Version="1.6.14" />
     <PackageReference Include="Microsoft.OpenApi.ApiManifest" Version="0.5.5-preview" />
     <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.6.14" />
-    <PackageReference Include="Microsoft.Plugins.Manifest" Version="0.0.6-preview" />
+    <PackageReference Include="Microsoft.Plugins.Manifest" Version="0.0.8-preview" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.Plugins.Manifest" Version="0.0.7-preview" />
```
Holding this at `0.0.7-preview` given that some properties are dropped in `0.0.8-preview`